src/test/java/seedu/guest/AppParametersTest.java

```diff
@@ -37,6 +39,15 @@ public void parse_invalidConfigPath_success() {
         assertEquals(expected, AppParameters.parse(parametersStub));
     }
 
+    @Test
+    public void equals() {
```

Comment:
Could we add another `assertFalse` for different types? Similar to `CommandResult` tests.

Reply:
Agreed. Added same values, different types and different configPath tests.